skema/skema-rs/mathml/src/ast/operator.rs

```diff
@@ -100,30 +121,28 @@ pub enum Operator {
     Arcsec,
     Arccsc,
     Arccot,
+    /// Mean operator
     Mean,
     /// Summation operator
-    Sum,
-    /// Summation operator with lowlimit and uplimit
-    SumUnderOver(SumUnderOver),
+    Summation(Summation),
+    /// Cross product operator
     Cross,
     /// Hat operator, e.g. \hat
     Hat,
     /// Hat operator with component, e.g. \hat{x}
     HatOp(HatOp),
-    /// Summation operator with lowlimit and uplimit
-    MsupDownArrow(MsupDownArrow),
     /// ↓ as an operator
-    DownArrow,
-    /// Definite Integral
-    Int,
-    /// Indefinite Integral with lowlimit and uplimit
-    MsubsupInt(MsubsupInt),
+    DownArrow(DownArrow),
+    /// Integrals
+    Int(Int),
+    /// Laplacian operator
     Laplacian,
     /// Closed surface integral operator --need to include explit dS integration variable when translating to latex
     SurfaceClosedInt,
     /// Closed surface integral operator -- doesn't need to include explicit dS integration variable when translating to latex
     /// E.g. \\oiint_S ∇ \cdot dS
     SurfaceClosedIntNoIntVar,
```

Comment:
`SurfaceClosedIntNoIntNoIntVar` should be combined with `SurfaceClosedInt`. Is there such a thing as an 'open' surface integral operator? If not, we may not need to specify 'closed' in the name of the operator.